tests/test_feedexport.py

```diff
@@ -1118,7 +1169,7 @@ def test_export_feed_export_fields(self):
                 {'egg': 'spam2', 'foo': 'bar2', 'baz': 'quux2'}
             ]
             yield self.assertExported(items, ['foo', 'baz', 'egg'], rows,
-                                      settings=settings, ordered=True)
+                                      settings=settings)
```

Comment:
Do we still need "ordered" parameter in assertExported / assertExportedCsv, or can we remove code for handling ordered=False?

Reply:
It seems so, https://github.com/scrapy/scrapy/pull/3696/commits/24f382fa459434cccfa4c0a8884a48d09d75e243 caused at least 3 tests to fail due to a different output order.

Reply:
Hmm… Looking in more details, maybe the issues are solvable without allowing a different order…